### ISSUE TITLE
stratis does not go online

### ISSUE THREAD

**Issue by fcbof** [stratisproject/stratisX#21]
hi there, my stratis wallett remains offline and 0 active connection. it is 41 weeks behind. can you tell what to do? thanks

**Comment by D-Code** [comment]
Had the same problem. I'm on Windows, btw.

Started with [version 2.0.0.2](https://github.com/stratisproject/stratisX/releases/tag/v2.0.0.2), and it got 1 connection, but it eventually got disconnected.

I exited the application and removed almost everything, except for wallet.dat (DO NOT DELETE wallet.dat), from %APPDATA%\Stratis. Then I downloaded and started [version 2.0.0](https://github.com/stratisproject/stratisX/releases/tag/v2.0.0). Now I didn't get any connections at all.

I exited the application - leaving %APPDATA%\Stratis as it was - and started version 2.0.0.2 again. Now I started getting connections after 2-3 minutes.

Is 2.0.0 required to be started before starting 2.0.0.2? Maybe it was a coincidence, I don't know; but those are the steps I took.
